samples/crypto/psa_tls/README.rst

```diff
@@ -149,6 +149,10 @@ After programming the sample to your development kit, complete the following ste
 
            openssl s_client -connect 192.0.2.1:4243 -cipher ECDHE-ECDSA-AES128-SHA256 -CAfile certs/ecdsa/root_cert.pem
 
+         .. note::
+
+            If the sample has been built with the DTLS overlay, add the ``-dtls`` option to the ``openssl`` command.
```

Comment:
Maybe add the code block showing the exact command, like it is done in other cases here.

Reply:
Added the code block

Reply:
The problem with the code block it is that it can be added both with EDCHE and RSA, but only EDCHE has an example of it's usage in here.

Reply:
If you have time, you could fix the code blocks so it's done for both? Otherwise, a JIRA can be created to fix RSA code blocks post NCS 2.8.0 release :)

Reply:
I think that it just gets out of hand very fast if we add every parameter combination as a separate code block. The information kind of gets buried unless you use tabs to hide non-relevant information.

Reply:
I agree with Markus. What if I add new tabs for DTLS?

Reply:
Let's try and see how that looks then :)